flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/TestFlinkTableSink.java

```diff
@@ -169,39 +154,6 @@ public void testOverwriteTable() throws Exception {
         icebergTable, Lists.newArrayList(SimpleDataUtil.createRecord(2, "b")));
   }
 
-  @TestTemplate
```

Comment:
moved these tests to the new `TestFlinkTableSinkExtended` which only tests on the dimensions of `isStreamingJob` boolean value. Currently every test run with 20+ combinations of parameters. Don't want to add range distribution test here to further slow down the test run time.